### PR TITLE
Show survey reminder call script in modal overlay

### PR DESCRIPTION
**Pull request by tyom**
[PT](https://www.pivotaltracker.com/story/show/87295520)

### CODE REVIEW

File: tests/angular-js/e2e/alternativeHelp.js [hunk 1]
```diff
@@ -14,7 +14,7 @@
   var modalHeading = modal.element(by.css('h2'));
   var modalSubmit = modal.element(by.css('button[type="submit"]'));
 
-  describe('alternativeHelp', function () {
+  ddescribe('alternativeHelp', function () {
```
Comment (axemonkey) [inline tests/angular-js/e2e/alternativeHelp.js]: Typo?

Comment (tyom) [inline tests/angular-js/e2e/alternativeHelp.js]: Oops. That's 'run this test only' keyword that shouldn't be there. Well spotted!

Comment (teneightfive) [inline tests/angular-js/e2e/alternativeHelp.js]: The JSHint gulp task should pick these up actually. We were tempted to have tests fail if JSHint failed at one point.

Comment (tyom) [inline tests/angular-js/e2e/alternativeHelp.js]: > The JSHint gulp task should pick these up actually. We were tempted to have tests fail if JSHint failed at one point.

I did. I just didn't notice.

File: tests/angular-js/e2e/providerFeedback.js [hunk 2]
```diff
@@ -217,7 +217,7 @@
 
   function do_assign () {
     var assignBtn = element(by.name('assign-provider'));
-    utils.scrollTo(assignBtn);
+    utils.scrollToBottom(assignBtn);
```
Comment (teneightfive) [inline tests/angular-js/e2e/providerFeedback.js]: :+1:

**Comment by teneightfive** [comment]
Looks good.

**Comment by tyom** [comment]
retest this please

**Comment by tyom** [comment]
retest this please

**Comment by tyom** [comment]
retest this please

**Comment by tyom** [comment]
retest this please

**Pull request merged**